app/android/app/src/com/salesforce/samples/myuserpicreactnative/MyUserPicReactNativeApp.java

```diff
@@ -28,14 +28,43 @@
 
 import android.app.Application;
 
+import com.facebook.react.ReactApplication;
+import com.facebook.react.ReactNativeHost;
+import com.facebook.react.ReactPackage;
+import com.facebook.react.shell.MainReactPackage;
+import com.imagepicker.ImagePickerPackage;
 import com.salesforce.androidsdk.app.SalesforceSDKManager;
 import com.salesforce.androidsdk.reactnative.app.SalesforceReactSDKManager;
 import com.salesforce.androidsdk.security.Encryptor;
 
+import java.util.Arrays;
+import java.util.List;
+
 /**
  * Application class for our application.
  */
-public class MyUserPicReactNativeApp extends Application {
+public class MyUserPicReactNativeApp extends Application implements ReactApplication {
+
+	private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
+		@Override
+		protected boolean getUseDeveloperSupport() {
+			return BuildConfig.DEBUG;
```

Comment:
Good catch, we don't. Fixed here: https://github.com/wmathurin/MyUserPicReactNative/pull/10/commits/1378d9f27337354893ae5ac3c0c3d642cb82613a